helpers/bump-global-version.sh

```diff
@@ -6,7 +6,7 @@ VERSION=$(grep '#*Lucet version ' Cargo.toml | sed 's/^ *# Lucet version *//')
 dry_run() {
     echo "Checking if the package can be published (dry run)..."
     echo
-    find lucetc lucet-* benchmarks/lucet-* -type f -maxdepth 1 -name 'Cargo.toml' -print | while read -r file; do
+    find lucetc lucet-* benchmarks/lucet-* lucet-runtime/lucet-*-type f -maxdepth 1 -name 'Cargo.toml' -print | while read -r file; do
```

Comment:
There is also a crate at lucet-idl/lucet-idl-test that we'd want to include.

Also, is there a space missing before `-type`?

Reply:
Indeed :)